README.md

```diff
@@ -2,9 +2,32 @@
 
 Operator to manage deadmanssnitch configs for Openshift Dedicated
 
+- [deadmanssnitch-operator](#deadmanssnitch-operator)
+  - [Overview](#overview)
+  - [Metrics](#metrics)
+  - [Alerts](#alerts)
+  - [Usage](#usage)
+
+## Overview
+
+The operator runs on hive. It has a single controller. It:
+- Requires a master Secret to talk to the Dead Man's Snitch API.
+  This secret is expected to be named `deadmanssnitch-api-key` and live in the `deadmanssnitch-operator` namespace.
+- Pays attention to ClusterDeployments that are:
```

Comment:
```suggestion
- Pays attention to [ClusterDeployments](https://github.com/openshift/hive/blob/master/config/crds/hive.openshift.io_clusterdeployments.yaml) that are:
```

Reply:
Funny you should say that. I'm looking at the [certman-operator README](https://github.com/openshift/certman-operator/blob/c6a0b0908211ee0c0763d63fc2fd1bce465e3dad/README.md#dependencies) which tried to do the same thing, but the link is now broken because hive moved the CRD.